docs/content/direct/wec-registration.md

```diff
@@ -1,5 +1,272 @@
 # Registering a Workload Execution Cluster
 
-This document will tell users what how to register a WEC in an ITS.
+This document explains how to register a Workload Execution Cluster (WEC) with an Inventory and Transport Space (ITS) in KubeStellar.
 
-TODO: write this.
+## Overview
+
+Registering a WEC with an ITS follows the same process as registering a managed cluster with an OCM hub. KubeStellar uses Open Cluster Management (OCM) for cluster registration and management.
+
+For the complete OCM registration process, refer to the [official Open Cluster Management documentation](https://open-cluster-management.io/docs/getting-started/installation/register-a-cluster/). The instructions below provide a complete registration process that is more focused on KubeStellar-specific considerations.
```

Comment:
The problem here is that it leads off with "For the complete OCM registration process, refer to ...". That in itself implies that what follows is not a complete registration process. That is not the distinction that you want to make. What follows _is_ a complete registration process. Do not start by implying otherwise. Start by saying what you mean. Even more importantly, don't start by talking about what is NOT coming. Start by talking about what _is_ coming. Finish with a reference to the _comprehensive_ OCM documentation. "comprehensive documentation" means something different from "complete process".